armi/reactor/blockParameters.py

```diff
@@ -161,6 +137,37 @@ def getBlockParameterDefinitions():
             categories=["cumulative"],
         )
 
+    with pDefs.createBuilder(
+        default=0.0, location=ParamLocation.VOLUME_INTEGRATED, categories=["depletion"]
+    ) as pb:
```

Comment:
Just to check. The default value for "save to database" here is "True".

That's what you want, right?

(I got no dog in that fight, I'm just making sure the implied default is what you want.)

Reply:
yup, I copied the context manager from the original context manager that was being used to define these parameters before I broke them out into their own thing. So this should be no change to the functionality as originally specified.